### PR TITLE
Change isVolumeConflict so read-only gce persistent disks don't confl…

### PR DESCRIPTION
**Pull request by runningwild**
…ict with each other

Fixes issue #8478.

This doesn't address the problem of two pods on different machines trying to mount the same disk in read-write mode, but I assume that is not within the scope of this issue.

### CODE REVIEW

File: plugin/pkg/scheduler/algorithm/predicates/predicates.go [hunk 1]
```diff
@@ -52,12 +52,13 @@ func (nodes ClientNodeInfo) GetNodeInfo(nodeID string) (*api.Node, error) {
 
 func isVolumeConflict(volume api.Volume, pod *api.Pod) bool {
```
Comment (thockin) [inline plugin/pkg/scheduler/algorithm/predicates/predicates.go]: Medium-to-long term, anything related to specific volume implementations should move to pkg/volume and be part of the plugin interface.

**Comment by erictune** [comment]
ok to test

**Comment by runningwild** [comment]
I don't really understand the error I'm seeing here, is this something I can fix?

**Comment by thockin** [comment]
Don't worry about EXPERIMENTAL JENKINS PR BUILDER.  He's experimental.

**Comment by davidopp** [comment]
LGTM

I think this is OK to merge because in some sense it was a bug (or at least a usability issue). (Otherwise we are under code freeze.)

**Pull request merged**